pkg/action/supervisor/action_supervisor.go

```diff
@@ -92,21 +97,22 @@ func (s *ActionSupervisor) checkMoveAction(action *turboaction.TurboAction) (boo
 		return false, err
 	}
 
-	if targetPod.Status.Phase != api.PodRunning {
-		err = fmt.Errorf("move-check failed: new pod status is %v", targetPod.Status.Phase)
-		glog.Error(err.Error())
-		return false, err
+	phase := targetPod.Status.Phase
+	if phase == api.PodRunning || (phase == api.PodPending && targetPod.DeletionGracePeriodSeconds == nil) {
+		moveDestination := moveSpec.Destination
+		actualHostingNode := targetPod.Spec.NodeName
+		if actualHostingNode == moveDestination {
+			glog.V(3).Infof("Move pod [%s/%s] action succeeded.", podNamespace, podName)
+			return true, nil
+		} else {
+			glog.Errorf("Move action failed. Incorrect move destination %s", actualHostingNode)
```

Comment:
1. how this can happen:  one case is that the desired node is down, and default scheduler moves the pod to another node.

2.whether to try again:  the environment (workloads, pods assignment, and cluster) may have changed since last move action, so it may not be right to move the pod again.  Will OpsMgr generate new recommendation for this Pod again?